Telemetry/CorePing.swift

```diff
@@ -57,6 +57,12 @@ class CorePing: TelemetryPing {
 
         let locale = NSBundle.mainBundle().preferredLocalizations.first!.stringByReplacingOccurrencesOfString("_", withString: "-")
 
+        let formatter = NSDateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd"
+        let date = formatter.stringFromDate(NSDate())
+
+        let timezoneOffset = NSTimeZone.localTimeZone().secondsFromGMT / 60
```

Comment:
fwiw, to test that on Android, I made the timezoneOffset getter into a utility function & added tests for it (as opposed to changing some values in code or the system settings & testing that it acted as expected). It can help make things better documented & less likely to regress in the future.